nss-day-cohort-41/react-nutshell-thepecans
# Issue #6

# Description
Implement articles per descriptions in tickets #6 and 7. I have additional style changes on another branch, to be pushed after this one.
Fixes #6 (full), #7 (partial)
## Type of change
# Testing Instructions
Navigate to ".../articles." Create new articles. Edit articles. Delete articles. If there are no friends' articles, make sure another user has articles saved to the database.
Like events, I had to change the timestamp format to sort and render correctly. The timestamp format is different from what was in the example database.
# Checklist:
